spec/jobs/mets_directory_scan_job_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
I wish I knew how to write better job specs.... I feel like these could be doing more for us, but I think that's Future Work.